application.py

```diff
@@ -1,44 +1,31 @@
-import os
 import requests
 from flask import Flask, render_template, request
-from werkzeug.utils import secure_filename
-
-ALLOWED_EXTENSIONS = {'png', 'jpg', 'jpeg'}
 
 app = Flask(__name__)
-app.config['UPLOAD_FOLDER'] = 'uploads/'
+app.config['IMAGE_PROCESSOR'] = 'http://whiteboardlivecoding-ocr.azurewebsites.net/api/upload_image'
 
 
 @app.route('/', methods=['GET', 'POST'])
 def index():
     if request.method == 'POST':
         file = request.files['file']
+
         if 'file' not in request.files or file.filename == '':
             return render_template('base.html', template='index.html')
-        elif file and allowed_file(file.filename):
-            # Save file to file system
-            filename = secure_filename(file.filename)
-            file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
-            filesFound = {'file': open(app.config['UPLOAD_FOLDER'] + filename, 'rb')}
-
-            r = requests.post('http://whiteboardlivecoding-ocr.azurewebsites.net/api/upload_image', files=filesFound)
+        elif file:
+            r = requests.post(app.config['IMAGE_PROCESSOR'], files={'file': file.read()})
             r = r.json()
```

Comment:
What happens if the request fails? Check status codes

Reply:
Ummm we don't really have a way to handle the backend failing and if it does then the request will throw an exception instead of returning a non-200 status code.

Reply:
A simple if statement to check if OK would probably be (Hehe pun) OK. Otherwise the call to json() will crash, and god knows what else will break

Reply:
True, fixed :)